cvat-ui/src/utils/hooks.ts

```diff
@@ -130,3 +133,9 @@ export function useAuthQuery(): Record<string, string> | null {
     const queryParams = new URLSearchParams(history.location.search);
     return authQuery(queryParams);
 }
+
+export function useRegisterShortcuts(shortcuts: Record<string, KeyMapItem>): void {
+    const store = getCVATStore();
+    const { registerShortcuts } = shortcutsActions;
+    store.dispatch(registerShortcuts(shortcuts));
+}
```

Comment:
That is not a hook, so:

- let's remove `use` in the beginning: `useRegisterShortcuts` -> `registerComponentShortcuts`
- let's move this function to `shortcuts-actions.ts`